Provided tests fail

I'm trying to create a RPM package of the 2.1.4 version, but the tests provided fail.

This is what I get:

```
guessit/test/test_options.py ........
guessit/test/test_yml.py F.....................
guessit/test/rules/processors_test.py ..
=================================== FAILURES ===================================
____________________________ TestYml.test[episodes] ____________________________
self = <guessit.test.test_yml.TestYml object at 0x7fdcf7ba0150>
filename = 'episodes.yml'
caplog = <pytest_capturelog.CaptureLogFuncArg object at 0x7fdcf7b96610>
    @pytest.mark.parametrize('filename', files, ids=ids)
    def test(self, filename, caplog):
        caplog.setLevel(logging.INFO)
        with open(os.path.join(__location__, filename), 'r', encoding='utf-8') as infile:
            data = yaml.load(infile, OrderedDictYAMLLoader)
        entries = Results()
    
        last_expected = None
        for string, expected in reversed(list(data.items())):
            if expected is None:
                data[string] = last_expected
            else:
                last_expected = expected
    
        default = None
        try:
            default = data['__default__']
            del data['__default__']
        except KeyError:
            pass
    
        for string, expected in data.items():
            TestYml.set_default(expected, default)
            entry = self.check_data(filename, string, expected)
            entries.append(entry)
>       entries.assert_ok()
guessit/test/test_yml.py:175: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = [Series/Californication/Season 2/Californication.2x05.Vaginatown.HDTV.XviD-0TV...spair-Paradise].Kono.Subarashii.Sekai.ni.Shukufuku.wo!.2.-..09.vostfr.FHD: OK!]
    def assert_ok(self):
        errors = [entry for entry in self if entry.error]
>       assert not errors
E       AssertionError: assert not [[PuyaSubs!] Digimon Adventure tri - 01 [720p][F9967949].mkv: ERROR! (valid=7, missing=1, different=0, extra=0, others...son 07\That '70s Show - S07E22 - 200...ears from Home.mkv: ERROR! (valid=7, missing=1, different=0, extra=0, others=0)]
guessit/test/test_yml.py:108: AssertionError
--------------------------------- Captured log ---------------------------------
test_yml.py                195 WARNING  [episodes.yml] Series/Californication/Season 2/Californication.2x05.Vaginatown.HDTV.XviD-0TV.avi: WARNING! (valid=9, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Series/dexter/Dexter.5x02.Hello,.Bandit.ENG.-.sub.FR.HDTV.XviD-AlFleNi-TeaM.[tvu.org.ru].avi: WARNING! (valid=12, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Series/Treme/Treme.1x03.Right.Place,.Wrong.Time.HDTV.XviD-NoTV.avi: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/Duckman/Duckman - S1E13 Joking The Chicken (unedited).avi: WARNING! (valid=5, extra=3)
test_yml.py                195 WARNING  [episodes.yml] Series/Simpsons/Saison 12 Fran\xe7ais/Simpsons,.The.12x08.A.Bas.Le.Sergent.Skinner.FR.avi: WARNING! (valid=6, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/Duckman/Duckman - 101 (01) - 20021107 - I, Duckman.avi: WARNING! (valid=6, extra=3)
test_yml.py                195 WARNING  [episodes.yml] Series/Futurama/Season 3 (mkv)/[\u2122] Futurama - S03E22 - Le chef de fer \xe0 30% ( 30 Percent Iron Chef ).mkv: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/The Office/Season 6/The Office - S06xE01.avi: WARNING! (valid=4, extra=2)
test_yml.py                195 WARNING  [episodes.yml] series/The Office/Season 4/The Office [401] Fun Run.avi: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/Mad Men Season 1 Complete/Mad.Men.S01E01.avi: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] series/Psych/Psych S02 Season 2 Complete English DVD/Psych.S02E02.65.Million.Years.Off.avi: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] series/Psych/Psych S02 Season 2 Complete English DVD/Psych.S02E03.Psy.Vs.Psy.Fran\xe7ais.srt: WARNING! (valid=9, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Series/Pure Laine/Pure.Laine.1x01.Toutes.Couleurs.Unies.FR.(Qu\xe9bec).DVB-Kceb.[tvu.org.ru].avi: WARNING! (valid=9, extra=3)
test_yml.py                195 WARNING  [episodes.yml] Series/Pure Laine/2x05 - Pure Laine - Je Me Souviens.avi: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/Tout sur moi/Tout sur moi - S02E02 - M\xe9nage \xe0 trois (14-01-2008) [Rip by Ampli].avi: WARNING! (valid=6, extra=3)
test_yml.py                195 WARNING  [episodes.yml] The.Mentalist.2x21.18-5-4.ENG.-.sub.FR.HDTV.XviD-AlFleNi-TeaM.[tvu.org.ru].avi: WARNING! (valid=11, extra=2)
test_yml.py                195 WARNING  [episodes.yml] series/__ Incomplete __/Dr Slump (Catalan)/Dr._Slump_-_003_DVB-Rip_Catalan_by_kelf.avi: WARNING! (valid=5, extra=4)
test_yml.py                195 WARNING  [episodes.yml] Series/Neverwhere/Neverwhere.05.Down.Street.[tvu.org.ru].avi: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/South Park/Season 4/South.Park.4x07.Cherokee.Hair.Tampons.DVDRip.[tvu.org.ru].avi: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/Kaamelott/Kaamelott - Livre V - Ep 23 - Le Forfait.avi: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/Duckman/Duckman - 110 (10) - 20021218 - Cellar Beware.avi: WARNING! (valid=6, extra=3)
test_yml.py                195 WARNING  [episodes.yml] Series/Breaking Bad/Minisodes/Breaking.Bad.(Minisodes).01.Good.Cop.Bad.Cop.WEBRip.XviD.avi: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/My Name Is Earl/My.Name.Is.Earl.S01Extras.-.Bad.Karma.DVDRip.XviD.avi: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] series/Freaks And Geeks/Season 1/Episode 4 - Kim Kelly Is My Friend-eng(1).srt: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /mnt/series/The Big Bang Theory/S01/The.Big.Bang.Theory.S01E01.mkv: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /media/Parks_and_Recreation-s03-e01.mkv: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /media/Parks_and_Recreation-s03-e02-Flu_Season.mkv: WARNING! (valid=5, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /media/Parks_and_Recreation-s03-x01.mkv: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /media/Parks_and_Recreation-s03-x02-Gag_Reel.mkv: WARNING! (valid=5, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /media/Band_of_Brothers-e01-Currahee.mkv: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /media/Band_of_Brothers-x02-We_Stand_Alone_Together.mkv: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /TV Shows/Mad.M-5x9.mkv: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /TV Shows/new.girl.117.hdtv-lol.mp4: WARNING! (valid=6, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Kaamelott - 5x44x45x46x47x48x49x50.avi: WARNING! (valid=4, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Example S01E01-02.avi: WARNING! (valid=4, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Example S01E01E02.avi: WARNING! (valid=4, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Series/Baccano!/Baccano!_-_T1_-_Trailer_-_[Ayu](dae8173e).mkv: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Series/Doctor Who (2005)/Season 06/Doctor Who (2005) - S06E01 - The Impossible Astronaut (1).avi: WARNING! (valid=6, extra=2)
test_yml.py                195 WARNING  [episodes.yml] The Sopranos - [05x07] - In Camelot.mp4: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] The.Office.(US).1x03.Health.Care.HDTV.XviD-LOL.avi: WARNING! (valid=9, extra=2)
test_yml.py                195 WARNING  [episodes.yml] /Volumes/data-1/Series/Futurama/Season 3/Futurama_-_S03_DVD_Bonus_-_Deleted_Scenes_Part_3.ogm: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Ben.and.Kate.S01E02.720p.HDTV.X264-DIMENSION.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /volume1/TV Series/Drawn Together/Season 1/Drawn Together 1x04 Requiem for a Reality Show.avi: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Sons.of.Anarchy.S05E06.720p.WEB.DL.DD5.1.H.264-CtrlHD.mkv: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /media/bdc64bfe-e36f-4af8-b550-e6fd2dfaa507/TV_Shows/Doctor Who (2005)/Saison 6/Doctor Who (2005) - S06E13 - The Wedding of River Song.mkv: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /mnt/videos/tvshows/Doctor Who/Season 06/E13 - The Wedding of River Song.mkv: WARNING! (valid=5, extra=1)
test_yml.py                195 WARNING  [episodes.yml] The.Simpsons.S24E03.Adventures.in.Baby-Getting.720p.WEB-DL.DD5.1.H.264-CtrlHD.mkv: WARNING! (valid=11, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /home/disaster/Videos/TV/Merlin/merlin_2008.5x02.arthurs_bane_part_two.repack.720p_hdtv_x264-fov.mkv: WARNING! (valid=13, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Da Vinci's Demons - 1x04 - The Magician.mkv: WARNING! (valid=5, extra=1)
test_yml.py                195 WARNING  [episodes.yml] CSI.S013E18.Sheltered.720p.WEB-DL.DD5.1.H.264.mkv: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Game of Thrones S03E06 1080i HDTV DD5.1 MPEG2-TrollHD.ts: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] gossip.girl.s01e18.hdtv.xvid-2hd.eng.srt: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Wheels.S03E01E02.720p.HDTV.x264-IMMERSE.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Wheels.S03E01-02.720p.HDTV.x264-IMMERSE.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Wheels.S03E01-E02.720p.HDTV.x264-IMMERSE.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Wheels.S03E01-04.720p.HDTV.x264-IMMERSE.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Marvels.Agents.of.S.H.I.E.L.D-S01E06.720p.HDTV.X264-DIMENSION.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Marvels.Agents.of.S.H.I.E.L.D.S01E06.720p.HDTV.X264-DIMENSION.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Marvels.Agents.of.S.H.I.E.L.D..S01E06.720p.HDTV.X264-DIMENSION.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Series/Friday Night Lights/Season 1/Friday Night Lights S01E19 - Ch-Ch-Ch-Ch-Changes.avi: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Dexter Saison VII FRENCH.BDRip.XviD-MiND.nfo: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Dexter Saison sept FRENCH.BDRip.XviD-MiND.nfo: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Pok\xe9mon S16 - E29 - 1280*720 HDTV VF.mkv: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] One.Piece.E576.VOSTFR.720p.HDTV.x264-MARINE-FORD.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Dexter.S08E12.FINAL.MULTi.1080p.BluRay.x264-MiND.mkv: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] One Piece - E623 VOSTFR HD [www.manga-ddl-free.com].mkv: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Falling Skies Saison 1.HDLight.720p.x264.VFF.mkv: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Sleepy.Hollow.S01E09.720p.WEB-DL.DD5.1.H.264-BP.mkv: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Sleepy.Hollow.S01E09.720p.WEB-DL.DD5.1.H.264-BS.mkv: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Battlestar.Galactica.S00.Pilot.FRENCH.DVDRip.XviD-NOTAG.avi: WARNING! (valid=9, extra=2)
test_yml.py                195 WARNING  [episodes.yml] The Big Bang Theory S00E00 Unaired Pilot VOSTFR TVRip XviD-VioCs: WARNING! (valid=9, extra=1)
test_yml.py                195 WARNING  [episodes.yml] 2.Broke.Girls.S03E10.480p.HDTV.x264-mSD.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] House.of.Cards.2013.S02E03.1080p.NF.WEBRip.DD5.1.x264-NTb.mkv: WARNING! (valid=12, extra=1)
test_yml.py                195 WARNING  [episodes.yml] House.of.Cards.2013.S02E03.1080p.Netflix.WEBRip.DD5.1.x264-NTb.mkv: WARNING! (valid=12, extra=1)
test_yml.py                195 WARNING  [episodes.yml] the.100.109.hdtv-lol.mp4: WARNING! (valid=6, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Criminal.Minds.5x03.Reckoner.ENG.-.sub.FR.HDTV.XviD-STi.[tvu.org.ru].avi: WARNING! (valid=11, extra=2)
test_yml.py                195 WARNING  [episodes.yml] 03-Criminal.Minds.avi: WARNING! (valid=3, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [Evil-Saizen]_Laughing_Salesman_14_[DVD][1C98686A].mkv: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Kaylith] Zankyou no Terror - 04 [480p][B4D4514E].mp4: WARNING! (valid=6, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [PuyaSubs!] Seirei Tsukai no Blade Dance - 05 [720p][32DD560E].mkv: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Doremi].Happiness.Charge.Precure.27.[1280x720].[DC91581A].mkv: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Daisei] Free!\uff1aIwatobi Swim Club - 01 ~ (BD 720p 10-bit AAC) [99E8E009].mkv: WARNING! (valid=9, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Tsundere] Boku wa Tomodachi ga Sukunai - 03 [BDRip h264 1920x1080 10bit FLAC][AF0C22CC].mkv: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [t.3.3.d]_Mikakunin_de_Shinkoukei_-_12_[720p][5DDC1352].mkv: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Anime-Koi] Sabagebu! - 06 [h264-720p][ABB3728A].mkv: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [aprm-Diogo4D] [BD][1080p] Nagi no Asukara 08 [4D102B7C].mkv: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Akindo-SSK] Zankyou no Terror - 05 [720P][Sub_ITA][F5CCE87C].mkv: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Naruto Shippuden Episode 366 VOSTFR.avi: WARNING! (valid=4, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Naruto Shippuden Episode 366v2 VOSTFR.avi: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [HorribleSubs] Ao Haru Ride - 06 [480p].mkv: WARNING! (valid=5, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [DeadFish] Tari Tari - 01 [BD][720p][AAC].mp4: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [NoobSubs] Sword Art Online II 06 (720p 8bit AAC).mp4: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [DeadFish] 01 - Tari Tari [BD][720p][AAC].mp4: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [NoobSubs] 06 Sword Art Online II (720p 8bit AAC).mp4: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [DeadFish] 12 - Tari Tari [BD][720p][AAC].mp4: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Something.Other.Season.1-3.avi: WARNING! (valid=3, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Something.Other.Season.1&3.avi: WARNING! (valid=3, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Something.Other.Season.1&3-1to12ep.avi: WARNING! (valid=3, extra=3)
test_yml.py                195 WARNING  [episodes.yml] FooBar 360: WARNING! (valid=3, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Real.Time.With.Bill.Maher.2014.10.31.HDTV.XviD-AFG.avi: WARNING! (valid=6, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Arrow.S03E21.Al.Sah-Him.1080p.WEB-DL.DD5.1.H.264-BS.mkv: WARNING! (valid=11, extra=1)
test_yml.py                195 WARNING  [episodes.yml] How to Make It in America - S02E06 - I'm Sorry, Who's Yosi?.mkv: WARNING! (valid=5, extra=1)
test_yml.py                195 WARNING  [episodes.yml] 24.S05E07.FRENCH.DVDRip.XviD-FiXi0N.avi: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] 12.Monkeys.S01E12.FRENCH.BDRip.x264-VENUE.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] The.Daily.Show.2015.07.01.Kirsten.Gillibrand.Extended.720p.CC.WEBRip.AAC2.0.x264-BTW.mkv: WARNING! (valid=12, extra=1)
test_yml.py                195 WARNING  [episodes.yml] The.Daily.Show.2015.07.01.Kirsten.Gillibrand.Extended.720p.ComedyCentral.WEBRip.AAC2.0.x264-BTW.mkv: WARNING! (valid=12, extra=1)
test_yml.py                195 WARNING  [episodes.yml] The.Daily.Show.2015.07.01.Kirsten.Gillibrand.Extended.720p.Comedy.Central.WEBRip.AAC2.0.x264-BTW.mkv: WARNING! (valid=12, extra=1)
test_yml.py                195 WARNING  [episodes.yml] The.Daily.Show.2015.07.01.Kirsten.Gillibrand.Extended.Interview.720p.CC.WEBRip.AAC2.0.x264-BTW.mkv: WARNING! (valid=11, extra=1)
test_yml.py                195 WARNING  [episodes.yml] The.Daily.Show.2015.07.02.Sarah.Vowell.CC.WEBRip.AAC2.0.x264-BTW.mkv: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] 90.Day.Fiance.S02E07.I.Have.To.Tell.You.Something.720p.HDTV.x264-W4F: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Doctor.Who.2005.S04E06.FRENCH.LD.DVDRip.XviD-TRACKS.avi: WARNING! (valid=10, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Astro.Le.Petit.Robot.S01E01+02.FRENCH.DVDRiP.X264.INT-BOOLZ.mkv: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Annika.Bengtzon.2012.E01.Le.Testament.De.Nobel.FRENCH.DVDRiP.XViD-STVFRV.avi: WARNING! (valid=9, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Dead.Set.02.FRENCH.LD.DVDRip.XviD-EPZ.avi: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Phineas and Ferb S01E00 & S01E01 & S01E02: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.2010.11.23.HDTV.XViD.Etc-Group: WARNING! (valid=5, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [HorribleSubs] Maria the Virgin Witch - 01 [720p].mkv: WARNING! (valid=5, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [ISLAND]One_Piece_679_[VOSTFR]_[V1]_[8bit]_[720p]_[EB7838FC].mp4: WARNING! (valid=9, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [ISLAND]One_Piece_679_[VOSTFR]_[8bit]_[720p]_[EB7838FC].mp4: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [Kaerizaki-Fansub]_One_Piece_679_[VOSTFR][HD_1280x720].mp4: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [Kaerizaki-Fansub]_One_Piece_679_[VOSTFR][FANSUB][HD_1280x720].mp4: WARNING! (valid=7, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [Kaerizaki-Fansub]_One_Piece_681_[VOSTFR][HD_1280x720]_V2.mp4: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [Kaerizaki-Fansub] High School DxD New 04 VOSTFR HD (1280x720) V2.mp4: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [Kaerizaki-Fansub] One Piece 603 VOSTFR PS VITA (960x544) V2.mp4: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [SGKK] Bleach 312v1 [720p/MKV]: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Ayako]_Infinite_Stratos_-_IS_-_07_[H264][720p][EB7838FC]: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Ayako] Infinite Stratos - IS - 07v2 [H264][720p][44419534]: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Bleach - s16e03-04 - 313-314: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Bleach.s16e03-04.313-314: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Bleach s16e03e04 313-314: WARNING! (valid=4, extra=1)
test_yml.py                195 WARNING  [episodes.yml] 003. Show Name - Ep Name.avi: WARNING! (valid=4, extra=3)
test_yml.py                195 WARNING  [episodes.yml] 003-004. Show Name - Ep Name.avi: WARNING! (valid=4, extra=3)
test_yml.py                195 WARNING  [episodes.yml] [ACX]_Wolf's_Spirit_001.mkv: WARNING! (valid=4, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Project.Runway.S14E00.and.S14E01.(Eng.Subs).SDTV.x264-[2Maverick].mp4: WARNING! (valid=8, extra=3)
test_yml.py                195 WARNING  [episodes.yml] [Hatsuyuki-Kaitou]_Fairy_Tail_2_-_16-20_[720p][10bit].torrent: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] [Hatsuyuki-Kaitou]_Fairy_Tail_2_-_16-20_(191-195)_[720p][10bit].torrent: WARNING! (valid=5, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Looney Tunes 1940x01 Porky's Last Stand.mkv: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] The.Good.Wife.S06E01.E10.720p.WEB-DL.DD5.1.H.264-CtrlHD/The.Good.Wife.S06E09.Trust.Issues.720p.WEB-DL.DD5.1.H.264-CtrlHD.mkv: WARNING! (valid=11, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Fear the Walking Dead - 01x02 - So Close, Yet So Far.REPACK-KILLERS.French.C.updated.Addic7ed.com.mkv: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Fear the Walking Dead - 01x02 - En Close, Yet En Far.REPACK-KILLERS.French.C.updated.Addic7ed.com.mkv: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] /av/unsorted/The.Daily.Show.2015.07.22.Jake.Gyllenhaal.720p.HDTV.x264-BATV.mkv: WARNING! (valid=8, extra=2)
test_yml.py                195 WARNING  [episodes.yml] [7.1.7.8.5] Foo Bar - 11 (H.264) [5235532D].mkv: WARNING! (valid=2, extra=6)
test_yml.py                197 ERROR    [episodes.yml] [PuyaSubs!] Digimon Adventure tri - 01 [720p][F9967949].mkv: ERROR! (valid=7, missing=1, different=0, extra=0, others=0)
test_yml.py                199 ERROR    [episodes.yml]     valid=7
test_yml.py                199 ERROR    [episodes.yml]         ('container', 'mkv')
test_yml.py                199 ERROR    [episodes.yml]         ('crc32', 'F9967949')
test_yml.py                199 ERROR    [episodes.yml]         ('episode', 1)
test_yml.py                199 ERROR    [episodes.yml]         ('release_group', 'PuyaSubs!')
test_yml.py                199 ERROR    [episodes.yml]         ('screen_size', '720p')
test_yml.py                199 ERROR    [episodes.yml]         ('title', 'Digimon Adventure tri')
test_yml.py                199 ERROR    [episodes.yml]         ('type', 'episode')
test_yml.py                199 ERROR    [episodes.yml]     missing=1
test_yml.py                199 ERROR    [episodes.yml]         ('mimetype', 'video/x-matroska')
test_yml.py                195 WARNING  [episodes.yml] Game.of.Thrones.S6.Ep5.X265.Dolby.2.0.KTM3.mp4: WARNING! (valid=9, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Game.of.Thrones.S03.1080p.BluRay.DTS-HD.MA.5.1.AVC.REMUX-FraMeSToR: WARNING! (valid=10, extra=1)
test_yml.py                197 ERROR    [episodes.yml] /series/The.B*.B*.T*.S10E01.1080p.HDTV.X264-DIMENSION[rarbg]/The.B*.B*.T*.S10E01.1080p.HDTV.X264-DIMENSION.mkv: ERROR! (valid=9, missing=1, different=0, extra=0, others=0)
test_yml.py                199 ERROR    [episodes.yml]     valid=9
test_yml.py                199 ERROR    [episodes.yml]         ('container', 'mkv')
test_yml.py                199 ERROR    [episodes.yml]         ('episode', 1)
test_yml.py                199 ERROR    [episodes.yml]         ('format', 'HDTV')
test_yml.py                199 ERROR    [episodes.yml]         ('release_group', 'DIMENSION[rarbg]')
test_yml.py                199 ERROR    [episodes.yml]         ('screen_size', '1080p')
test_yml.py                199 ERROR    [episodes.yml]         ('season', 10)
test_yml.py                199 ERROR    [episodes.yml]         ('title', 'The B B T')
test_yml.py                199 ERROR    [episodes.yml]         ('type', 'episode')
test_yml.py                199 ERROR    [episodes.yml]         ('video_codec', 'h264')
test_yml.py                199 ERROR    [episodes.yml]     missing=1
test_yml.py                199 ERROR    [episodes.yml]         ('mimetype', 'video/x-matroska')
test_yml.py                195 WARNING  [episodes.yml] Vice.News.Tonight.2016.10.10.1080p.HBO.WEBRip.AAC2.0.H.264-monkee: WARNING! (valid=9, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S15E18.Nice.Title.720p.AE.WEBRip.AAC2.0.H.264-monkee: WARNING! (valid=12, extra=1)
test_yml.py                197 ERROR    [episodes.yml] Show.Name.2013.S02E03.1080p.NF.WEBRip.DD5.1.x264-NTb.mkv: ERROR! (valid=13, missing=1, different=0, extra=0, others=0)
test_yml.py                199 ERROR    [episodes.yml]     valid=13
test_yml.py                199 ERROR    [episodes.yml]         ('title', 'Show Name')
test_yml.py                199 ERROR    [episodes.yml]         ('year', 2013)
test_yml.py                199 ERROR    [episodes.yml]         ('season', 2)
test_yml.py                199 ERROR    [episodes.yml]         ('episode', 3)
test_yml.py                199 ERROR    [episodes.yml]         ('screen_size', '1080p')
test_yml.py                199 ERROR    [episodes.yml]         ('streaming_service', 'Netflix')
test_yml.py                199 ERROR    [episodes.yml]         ('format', 'WEBRip')
test_yml.py                199 ERROR    [episodes.yml]         ('audio_codec', 'AC3')
test_yml.py                199 ERROR    [episodes.yml]         ('audio_channels', '5.1')
test_yml.py                199 ERROR    [episodes.yml]         ('video_codec', 'h264')
test_yml.py                199 ERROR    [episodes.yml]         ('release_group', 'NTb')
test_yml.py                199 ERROR    [episodes.yml]         ('container', 'mkv')
test_yml.py                199 ERROR    [episodes.yml]         ('type', 'episode')
test_yml.py                199 ERROR    [episodes.yml]     missing=1
test_yml.py                199 ERROR    [episodes.yml]         ('mimetype', 'video/x-matroska')
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S01.1080p.CR.WEBRip.AAC.2.0.x264-monkee: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.-.07.(2016).[RH].[English.Dubbed][WEBRip]..[HD.1080p]: WARNING! (valid=8, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S01E02.Exfil.1080p.USAN.WEBRip.AAC2.0.x264-AJP69: WARNING! (valid=11, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S01E01.720p.ID.WEBRip.AAC2.0.x264-BTW: WARNING! (valid=11, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Proof.2015.S01E10.1080p.WEB-DL.DD5.1.H.264-KINGS.mkv: WARNING! (valid=11, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /tv/Daniel Tiger's Neighborhood/S02E06 - Playtime Is Different.mp4: WARNING! (valid=6, extra=1)
test_yml.py                195 WARNING  [episodes.yml] The.Messengers.2015.S01E07.1080p.WEB-DL.DD5.1.H264.Nlsubs-Q/QoQ-sbuSLN.462.H.1.5DD.LD-BEW.p0801.70E10S.5102.sregnesseM.ehT.mkv: WARNING! (valid=13, extra=1)
test_yml.py                195 WARNING  [episodes.yml] /Finding.Carter.S02E01.Love.the.Way.You.Lie.1080p.WEB-DL.AAC2.0.H.264-NL/LN-462.H.0.2CAA.LD-BEW.p0801.eiL.uoY.yaW.eht.evoL.10E20S.retraC.gnidniF.mkv: WARNING! (valid=12, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Mr.Robot.S02E12.1080p.WEB-DL.DD5.1-NL.Subs-Het.Robot.Team.OYM/sbuS LN-1.5DD LD-BEW p0801 21E20S toboR .rM.mkv: WARNING! (valid=9, extra=4)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S06E03.1080p.HDTV.Legendado: WARNING! (valid=2, extra=5)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S06E05.1080p.WEBRip.Legendado.PT: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S04E21.Aint.Nothing.Like.the.Real.Thing.German.Custom.Subbed.720p.HDTV.x264.iNTERNAL-BaCKToRG: WARNING! (valid=9, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S01.Season.Complet.WEBRiP.Ro.Subbed.TM: WARNING! (valid=6, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.(2013).Season.3.-.Eng.Soft.Subtitles.720p.WEBRip.x264.[MKV,AC3,5.1].Ehhhh: WARNING! (valid=12, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S06E05.Das.Toor.German.AC3.Dubbed.HDTV.German: WARNING! (valid=7, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.S01E05.3xpl0its.wmv.720p.WEBdl.EN-SUB.x264-[MULVAcoded].mkv: WARNING! (valid=9, extra=2)
test_yml.py                195 WARNING  [episodes.yml] Show.Name.1x01.eps1.0.hellofriend.(HDiTunes.Ac3.Esp).(2015).By.Malaguita.avi: WARNING! (valid=11, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Star.Wars.Rebels.S01E01.Spark.of.Rebellion.ALTERNATE.CUT.HDTV.x264-W4F.mp4: WARNING! (valid=10, extra=1)
test_yml.py                195 WARNING  [episodes.yml] Dr.Ken.S01E21.SAMPLEFIX.720p.HDTV.x264-SVA: WARNING! (valid=9, extra=1)
test_yml.py                197 ERROR    [episodes.yml] D:\\TV\\SITCOMS (CLASSIC)\\That '70s Show\\Season 07\\That '70s Show - S07E22 - 2000 Light Years from Home.mkv: ERROR! (valid=7, missing=1, different=0, extra=0, others=0)
test_yml.py                199 ERROR    [episodes.yml]     valid=7
test_yml.py                199 ERROR    [episodes.yml]         ('title', "That '70s Show")
test_yml.py                199 ERROR    [episodes.yml]         ('season', 7)
test_yml.py                199 ERROR    [episodes.yml]         ('episode', 22)
test_yml.py                199 ERROR    [episodes.yml]         ('episode_title', '2000 Light Years from Home')
test_yml.py                199 ERROR    [episodes.yml]         ('other', 'Classic')
test_yml.py                199 ERROR    [episodes.yml]         ('container', 'mkv')
test_yml.py                199 ERROR    [episodes.yml]         ('type', 'episode')
test_yml.py                199 ERROR    [episodes.yml]     missing=1
test_yml.py                199 ERROR    [episodes.yml]         ('mimetype', 'video/x-matroska')
test_yml.py                195 WARNING  [episodes.yml] Series_name.2005.211.episode.title.avi: WARNING! (valid=7, extra=1)
=============================== warnings summary ===============================
:171
  'pytest_runtest_makereport' hook uses deprecated __multicall__ argument
None
  pytest_funcarg__caplog: declaring fixtures using "pytest_funcarg__" prefix is deprecated and scheduled to be removed in pytest 4.0.  Please remove the prefix and use the @pytest.fixture decorator instead.
  pytest_funcarg__capturelog: declaring fixtures using "pytest_funcarg__" prefix is deprecated and scheduled to be removed in pytest 4.0.  Please remove the prefix and use the @pytest.fixture decorator instead.
-- Docs: http://doc.pytest.org/en/latest/warnings.html
========= 1 failed, 63 passed, 4 skipped, 3 warnings in 16.22 seconds ==========
```

--- a/guessit/test/episodes.yml
+++ b/guessit/test/episodes.yml
@@ -1681,7 +1681,6 @@
   episode_title: My Perspective
   source: DVD
   other: Rip
-  mimetype: video/x-msvideo
   release_group: WAT
   season: 6
   title: Scrubs
@@ -1691,7 +1690,6 @@
 : container: mkv
   crc32: F9967949
   episode: 1
-  mimetype: video/x-matroska
   release_group: PuyaSubs!
   screen_size: 720p
   title: Digimon Adventure tri
@@ -1708,7 +1706,6 @@
   episode: 7
   episode_title: Michael B Jordan
   source: HDTV
-  mimetype: video/x-msvideo
   other: Proper
   proper_count: 1
   release_group: W4F
@@ -1870,7 +1867,6 @@
   container: mp4
   episode: 1
   source: HDTV
-  mimetype: video/mp4
   release_group: k3n
   season: 2
   title: Fear The Walking Dead
@@ -2072,7 +2068,6 @@
   other: Rip
   release_group: qqss44
   container: avi
-  mimetype: video/x-msvideo
   type: episode
 
 # WEBCap
@@ -2116,7 +2111,6 @@
 : container: mkv
   episode: 1
   source: HDTV
-  mimetype: video/x-matroska
   release_group: DIMENSION
   screen_size: 1080p
   season: 10
@@ -3359,7 +3353,6 @@
   episode: 22
   episode_title: 2000 Light Years from Home
   container: mkv
-  mimetype: video/x-matroska
   type: episode
 
 ? Show.Name.S02E01.Super.Title.720p.WEB-DL.DD5.1.H.264-ABC.nzb

--- a/guessit/test/streaming_services.yaml
+++ b/guessit/test/streaming_services.yaml
@@ -246,7 +246,6 @@
   video_codec: H.264
   release_group: NTb
   container: mkv
-  mimetype: video/x-matroska
   type: episode
 
 # Streaming service: CBS

--- a/guessit/test/various.yml
+++ b/guessit/test/various.yml
@@ -545,7 +545,6 @@
   - 1
   - 7
   episode_title: FooBar-Group  # Make sure it doesn't conflict with uuid
-  mimetype: video/x-msvideo
   season: 1
   title: Test
   type: episode
@@ -901,4 +900,3 @@
   video_bit_rate: 1.5Mbps
   container: mp4
   type: movie
-

